### ISSUE TITLE
The game board is upside down 

### ISSUE THREAD

**Issue by nkmcalli** [buttonmen-dev/buttonmen#2435]
Periodically I try and use my opponents dice to capture my own dice.  If we were sitting at a table playing in real life, my dice would be in front of me, and my opponents dice would be across from me.  On the site we have this upside down.  My dice are across from me, and my opponents dice are in front of me.  Why?  Can we turn the board back around?

**Comment by blackshadowshade** [comment]
Huh, interesting!

So, if we did this, we'd put in a user preference to allow the play mats to be swapped, meaning that your button would be on the right and your dice would be at the bottom. I presume that's what you're asking for?

**Comment by irilyth** [comment]
Would you want your button's image on the left, in that case, or the right?

**Comment by nkmcalli** [comment]
I don't think the side of the button image matters as much.  I would choose left because I'm used to reading left to right, so I look to the left to start, I guess.

**Comment by danlangford** [comment]
I agree and was I suggested this one time and was met with resistance. The conversation may be of interest. I’ll try to find the forum post 

https://www.buttonweavers.com/ui/forum.html#!threadId=235

**Comment by TheOrgg** [comment]
This would be cool, but only if it was an optional setting.
